pass-nihms-loader/nihms-data-harvest/pom.xml

```diff
@@ -90,13 +90,13 @@
   </dependencies>
 
   <build>
+    <finalName>nihms-data-harvest-cli</finalName>
     <plugins>
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
         <configuration>
           <mainClass>org.eclipse.pass.loader.nihms.NihmsHarvesterCLI</mainClass>
-          <finalName>nihms-data-harvest-cli</finalName>
```

Comment:
I think this needs to remain here.

Reply:
For this one I was getting compilation errors. I think there was a recent mvn change or something. I will put it back in to get the error.

Reply:
Done. This has been reverted, but will be looked into in the future, per our discussion.